daemon/algod/api/server/router.go

```diff
@@ -103,6 +104,9 @@ func NewRouter(logger logging.Logger, node *node.AlgorandFullNode, shutdown <-ch
 	e.Listener = listener
 	e.HideBanner = true
 
+	// Make sure '/v1/ledger/supply' and '/v1/ledger/supply/' are both accepted.
+	e.Pre(middleware.RemoveTrailingSlash())
+
```

Comment:
Which one of these is the correct one according to the REST API ? I'd guess the former, and so the Java SDK would need to be modified to emit the correct request.

Also, note that the comment suggest that it applies to `/v1/ledger/supply` command where is it affects all the v1 endpoints.

Reply:
The correct form is with no trailing slash. The Java SDK should also be fixed to stop appending the trailing slash.

The fix that I added here effects all endpoints, they'll now be accepted with and without the trailing slash.